panel/src/components/Forms/FormButtons.vue

```diff
@@ -180,7 +180,7 @@ export default {
           // If setting lock failed, a competing lock has been set between
           // API calls. In that case, discard changes, stop setting lock and
           // listen to concurrent lock
-          this.$store.dispatch("form/revert", this.id);
+          this.$store.dispatch("content/revert", this.id);
```

Comment:
Do we get the full id here? Shouldn't it be `"pages/" + this.id`

Reply:
Since `this.id` is computed for `this.$store.state.content.current` it should already hold the full id. Keep in mind that `FormButtons.vue` is used for users and files as well.

Reply:
Ahh, that makes sense.